partner_compassion/views/partner_compassion_view.xml

```diff
@@ -348,9 +325,9 @@
         <field name="view_id" ref="account.invoice_tree"/>
         <field name="domain">[('type','in', ['out_invoice', 'out_refund'])]</field>
         <field name="context">{
-            'search_default_partner_id': active_id,
-            'default_partner_id': active_id
-            }
+                    'search_default_partner_id': active_id,
+                    'default_partner_id': active_id
```

Comment:
This change is producing your bug in Travis. It is probably due to a wrong merge during rebase.